src/aggregator/aggregator/entry.go

```diff
@@ -685,17 +693,6 @@ func (e *Entry) addTimed(
 		return errEntryClosed
 	}
 
-	// Reject datapoints that arrive too late or too early.
-	if err := e.checkTimestampForTimedMetric(
-		metric,
-		currTime.UnixNano(),
-		metadata.StoragePolicy.Resolution().Window,
```

Comment:
bug: this was always zero for `AddTimedWithStagedMetadatas` since `metadata` is empty.